__docs__/wonder-blocks-popover/popover-content.stories.tsx

```diff
@@ -106,8 +108,9 @@ export const WithIcon: StoryComponentType = {
     args: {
         title: "Popover with Icon",
         content: "Popovers can include images on the left.",
-        icon: <img src="/logo.svg" width="100%" alt="Wonder Blocks logo" />,
+        icon: <img src="./logo.svg" width="100%" alt="Wonder Blocks logo" />,
```

Comment:
**note:** Adding relative paths to fix some issues in the production version of SB (these images are broken in there).